packages/block-library/src/image/index.php

```diff
@@ -307,6 +310,12 @@ class="wp-lightbox-overlay zoom"
 				<button type="button" aria-label="$close_button_label" style="fill: $close_button_color" class="close-button">
 					<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="20" height="20" aria-hidden="true" focusable="false"><path d="m13.06 12 6.47-6.47-1.06-1.06L12 10.94 5.53 4.47 4.47 5.53 10.94 12l-6.47 6.47 1.06 1.06L12 13.06l6.47 6.47 1.06-1.06L13.06 12Z"></path></svg>
 				</button>
+				<button type="button" aria-label="$prev_button_label" style="fill: $close_button_color" class="prev-button" data-wp-bind--hidden="!state.hasNavigation" data-wp-on--click="actions.showPreviousImage" data-wp-bind--disabled="!state.hasPreviousImage">
```

Comment:
It needs to be further validate, but it's likely that we will have to set `aria-disabled` instead of `disabled` so the button doesn't loose focus when there are no more items to navigate to with the keyboard interactions using `space` or `enter`.

Reply:
Yes, I confirmed that in my testing. As soon as button gets into `disabled` state the focus moves to the `body` element. In general, tabable elements should always be in the same order so it is predictable. Inactive buttons should be present but announce their disabled state.

I'm also not entirely sure how it all should work with arrow right and left when focus is inside the lightbox on the close button or arrow keys. I would appreciate some hepl from accessibility experts here.